server/tests/client.ts

```diff
@@ -47,8 +51,19 @@ describe('Test a client controllers', function () {
   before(async function () {
     this.timeout(120000)
 
-    server = await flushAndRunServer(1)
-    server.accessToken = await serverLogin(server)
+    servers = await flushAndRunMultipleServers(2)
+    const server = servers[0]
+
+    await setAccessTokensToServers(servers)
+
+    {
+      const res = await uploadVideo(servers[0].url, servers[0].accessToken, { name: 'video' })
```

Comment:
Do we really need to upload a video here? According to line 78 there's already a video on the server.

If we need this line, then line 78 has to be updated (that's why tests are failing).